mmv1/products/tags/TagBinding.yaml

```diff
@@ -50,7 +50,7 @@ examples:
       org_id: :ORG_ID
   - !ruby/object:Provider::Terraform::Examples
     name: 'tag_binding_with_namespaced_value'
-    skip_test: true
+    skip_test: false
```

Comment:
```suggestion
```
we can remove it then. It should be default to false

Reply:
Appreciate the feedback, but just added this change a couple minutes ago.

Reply:
Updated to remove it.